File: car-lease-service/src/main/java/org/jesperancinha/car/lease/dto/LeaseDto.java
package org.jesperancinha.car.lease.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.jesperancinha.car.lease.model.Car;
import org.jesperancinha.car.lease.model.Customer;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class LeaseDto {

    private Long id;

    private CarDto car;

    private CustomerDto customer;

    private Long millage;

    private Long duration;

    private Long interestRate;

    private Long netPrice;

    private Double leaseRate;
}

File: car-lease-service/src/main/kotlin/org/jesperancinha/car/lease/dto/LeaseDto.kt
package org.jesperancinha.car.lease.dto

import lombok.AllArgsConstructor
import lombok.Builder
import lombok.Data
import lombok.NoArgsConstructor

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
class LeaseDto {
    private val id: Long? = null
    private val carId: Long? = null
    private val customerId: Long? = null
    private val duration: Long? = null
    private val interestRate: Long? = null
    private val leaseRate: Double? = null
}